src/meta/src/manager/catalog/mod.rs

```diff
@@ -2192,6 +2194,38 @@ impl CatalogManager {
         Ok(())
     }
 
+    pub async fn create_comment(&self, comment: Comment) -> MetaResult<NotificationVersion> {
+        let core = &mut *self.core.lock().await;
+        let database_core = &mut core.database;
+        database_core.ensure_table_view_or_source_id(&comment.table_id)?;
+
+        let mut tables = BTreeMapTransaction::new(&mut database_core.tables);
+
+        // TODO: dont unwrap
+        let mut table = tables.get_mut(comment.table_id).unwrap();
+        let col_idx = comment.column_index;
+        if col_idx > 0 {
```

Comment:
A little strange.
In PostgreSQL, when `objoid` (`comment.table_id` ici) refer to a table, the `objsubid` eq `0` means table itself, otherwise `column number`. But in RisingWave, we have `_row_id`.